ui/app/AppLayouts/Profile/Sections/NotificationsContainer.qml

```diff
@@ -12,7 +12,72 @@ ScrollView {
     contentHeight: notificationsContainer.height
     clip: true
 
-    Rectangle {
+    ModalPopup {
```

Comment:
A cool trick is to put this in a `Component` like this https://github.com/status-im/nim-status-client/blob/master/ui/app/AppLayouts/Browser/BrowserLayout.qml#L73
And then you create the object like so: https://github.com/status-im/nim-status-client/blob/master/ui/app/AppLayouts/Browser/BrowserLayout.qml#L200
And in the `onClosed` of the modal, you put `destroy()` so that it frees it's memory like so https://github.com/status-im/nim-status-client/blob/master/ui/app/AppLayouts/Browser/BrowserConnectionModal.qml#L39

All of this is so that we don't create the Modal on app start, so it's faster to start and doesn't use more memory than needed